src/assets/Azure.Core.Shared/OperationPoller.cs

```diff
@@ -15,51 +17,52 @@ namespace Azure.Core
     /// </summary>
     internal sealed class OperationPoller
     {
-        private readonly DelayStrategyInternal _delayStrategy;
+        private readonly DelayStrategy _delayStrategy;
```

Comment:
Curious: Are LRO changes required for sample updates?

Reply:
Build will fail without this update. I can create a separate PR for it.

Reply:
In fact, I think I must do it.

Reply:
https://github.com/Azure/autorest.csharp/pull/3281 - @JoshLove-msft already created a PR